src/flamenco/snapshot/fd_snapshot_http.c

```diff
@@ -44,6 +44,7 @@ fd_snapshot_http_t *
 fd_snapshot_http_new( void *               mem,
                       uint                 dst_ipv4,
                       ushort               dst_port,
+                      ulong                base_slot,
```

Comment:
I suggest moving this to 'set path'.

Reply:
set_path happens deep in the api stack where base_slot isn't available